Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Docker image for installing dependencies on Linux and running tests.
+# Build with:
+# docker build --tag=gerardpuig/ubuntu-cleaner --file=dockerfiles/Dockerfile-linux .
+# Run with:
+# docker run -it --rm gerardpuig/ubuntu-cleaner make test
+# Or for interactive shell:
+# docker run -it --rm gerardpuig/ubuntu-cleaner
+# For running UI:
+# xhost +"local:docker@"
+# docker run -e DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --it -rm gerardpuig/ubuntu-cleaner ubuntu-cleaner
+FROM ubuntu:18.04
```

Comment:
This is Ubuntu LTS, because that's what we should at least support

Reply:
yep, it actually works on 19.04 but indeed the LTS are the official to be supported